elasticsearch/esql/functions.py

```diff
@@ -649,7 +649,7 @@ def min_over_time(field: ExpressionType) -> InstrumentedExpression:
 
 
 def multi_match(
-    query: ExpressionType, fields: ExpressionType, options: ExpressionType = None
+    query: ExpressionType, *fields: ExpressionType, options: ExpressionType = None
```

Comment:
I agree that this is nicer, but I think it's a breaking change?

Reply:
Will commit this change separately, so that we have a log entry to include in the release notes.